### PR TITLE
Django empty project structure [PVD-XXX]

### PR DESCRIPTION
**Pull request by pawlew2k**
django service

### CODE REVIEW

File: backend/mysite/mysite/settings.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    },
+]
+
+
+# Internationalization
+# https://docs.djangoproject.com/en/4.0/topics/i18n/
+
+LANGUAGE_CODE = 'en-us'
+
+TIME_ZONE = 'UTC'
```
Comment (piotrkica) [inline backend/mysite/mysite/settings.py:109]: replace timezone to CET

**Review by piotrkica — CHANGES_REQUESTED** [review]
Overall good job, but fix that one line

File: backend/mysite/mysite/settings.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-x+#^39p!33-aaql66y7qbjz(@k-c36k0o50c@c)a14g)5^4(6x'
```
Comment (Goader) [inline backend/mysite/mysite/settings.py:23]: What is that key for? And shouldn't it be hidden somewhere? For example, in `.env`

File: backend/mysite/mysite/settings.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```
Comment (Goader) [inline backend/mysite/mysite/settings.py:26]: It's better to read this from the environment variable for easy configuration in production

File: backend/mysite/mysite/settings.py [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+WSGI_APPLICATION = 'mysite.wsgi.application'
+
+
+# Database
+# https://docs.djangoproject.com/en/4.0/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
```
Comment (Goader) [inline backend/mysite/mysite/settings.py:79]: We use MongoDB, so we should look for the engine explicitly for Mongo and specify it here

**Review by Goader — CHANGES_REQUESTED** [review]
Mostly good, configuration needs a little refactoring

**Comment by piotrkica** [comment]
Replace "mysite" occurrences (both directory names and docstrings) to "planty"

**Pull request merged**